cibuildwheel/macos.py

```diff
@@ -151,16 +146,23 @@ def build(project_dir, output_dir, test_command, before_test, test_requires, tes
             print("cibuildwheel: python available on PATH doesn't match our installed instance. If you have modified PATH, ensure that you don't overwrite cibuildwheel's entry or insert python above it.", file=sys.stderr)
             exit(1)
 
+        dependency_constraint_flags = []
+        if dependency_constraints:
+            python_version, _, _ = config.version.partition('-')
```

Comment:
Blurgh. Maybe we should split the PyPy `version` fields into `version` and `pypy_version` (if applicable)?
PyPy does something similar:

```
>>>> sys.version_info
(major=3, minor=6, micro=9, releaselevel='final', serial=0)
>>>> sys.pypy_version_info
(major=7, minor=3, micro=0, releaselevel='final', serial=0)
```

Reply:
(Again, nothing urgent, but if everything else is perfect, and this catches my eye, I can just as well mention it ;-) )

Reply:
If not, maybe we should make the versions on Linux consistent.

Reply:
No, it's a good point! I like your suggestion, I'll see if it's feasible.

Reply:
done!